libvast/vast/detail/env.hpp

```diff
@@ -0,0 +1,26 @@
+//    _   _____   __________
+//   | | / / _ | / __/_  __/     Visibility
+//   | |/ / __ |_\ \  / /          Across
+//   |___/_/ |_/___/ /_/       Space and Time
+//
+// SPDX-FileCopyrightText: (c) 2021 The VAST Contributors
+// SPDX-License-Identifier: BSD-3-Clause
+
+#pragma once
+
+#include <string>
+
+namespace vast::detail {
+
+/// A thread-safe wrapper around `::getenv`.
+/// @param var The environment variable.
+/// @returns The environment variables contents, or an empty string if the
+/// variable is unset.
+std::string env(const char* var);
```

Comment:
Returning `std::optional<std::string>` would make it possible to distinguish between an unset and a set variable that is empty.

Reply:
Done.